api/internal/managers/linux/infra/install.go

```diff
@@ -29,7 +29,12 @@ import (
 	kyaml "sigs.k8s.io/yaml"
 )
 
-const K0sComponentName = "Runtime"
+const (
+	// Skip running the KOTS app preflights in the Admin Console; we run them in the manager experience installer when ENABLE_V3 is enabled
+	SkipPreflights = true
```

Comment:
nit: no need to define a global var, just set it to true in the options and add a comment there